pipeline.yml

```diff
@@ -880,10 +879,9 @@ jobs:
       resource: masterbosh-state
   - task: decrypt-private-key
     file: pipeline-tasks/decrypt.yml
-    config:
-      inputs:
-      - name: pipeline-tasks
-      - name: common-masterbosh
+    input_mapping:
+    - name: pipeline-tasks
```

Comment:
Actually, why is this changing in the first place? The original config looks correct--we're overriding the config in pipeline-tasks by adding a second input.